src/ImageSharp/Formats/Tiff/TiffFrameMetadataResolutionExtensions.cs

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+
+using SixLabors.ImageSharp.Common.Helpers;
+using SixLabors.ImageSharp.Metadata;
+using SixLabors.ImageSharp.Metadata.Profiles.Exif;
+
+namespace SixLabors.ImageSharp.Formats.Experimental.Tiff
+{
+    internal static class TiffFrameMetadataResolutionExtensions
```

Comment:
This class shouldn't exist. Instead `UnitConverter` should be extended to allow setting the values in a given ExifProfile and reused for both jpeg and Tiff.

Reply:
I'll make it in the coming days

Reply:
@IldarKhayrutdinov thanks for your help, its highly appreciated!

Reply:
please review #1570